pkgs/development/libraries/glibc/common.nix

```diff
@@ -92,6 +92,11 @@ stdenv.mkDerivation ({
         url = "https://salsa.debian.org/glibc-team/glibc/raw/49767c9f7de4828220b691b29de0baf60d8a54ec/debian/patches/localedata/locale-C.diff";
         sha256 = "0irj60hs2i91ilwg5w7sqrxb695c93xg0ik7yhhq9irprd7fidn4";
       })
+      (fetchurl {
+        url = "https://bugs.debian.org/cgi-bin/bugreport.cgi?att=1;bug=874160;filename=0001-Default-to-C.UTF-8-on-setlocale-.-if-no-env-vars-are.patch;msg=5";
+        name = "0001-Default-to-C.UTF-8-on-setlocale-.-if-no-env-vars-are.patch";
```

Comment:
So debian switched to `C.UTF-8` now?

Reply:
No this is just a proposed patch

Reply:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=874160